dashboard/src/containers/AppUpgradeContainer/AppUpgradeContainer.tsx

```diff
@@ -47,12 +50,12 @@ function mapDispatchToProps(dispatch: ThunkDispatch<IStoreState, null, Action>)
     checkChart: (repo: string, chartName: string) =>
       dispatch(actions.repos.checkChart(repo, chartName)),
     clearRepo: () => dispatch(actions.repos.clearRepo()),
-    fetchChartVersions: (id: string) => dispatch(actions.charts.fetchChartVersions(id)),
+    fetchChartVersions: (namespace: string, id: string) => dispatch(actions.charts.fetchChartVersions(namespace, id)),
     fetchRepositories: () => dispatch(actions.repos.fetchRepos()),
     getAppWithUpdateInfo: (releaseName: string, ns: string) =>
```

Comment:
here you can see that there are other functions that require the namespace as well: checkChart, fetchRepositories, getAppWithUpdateInfo (that last one is used in many places).

Reply:
Updated `checkChart` and `fetchRepositories`, though `getAppWithUpdateInfo` already included the namespace (just hidden with a 2-letter var ns, I updated it and put the namespace arg first for consistency).

Need to test the built images IRL, I suspect I'll need to change the inputs for the SelectRepo component.